api/default.go

```diff
@@ -3,10 +3,10 @@ package api
 var DefaultListLimit = 30
 
 //IsValidToken checks if a token provided is valid.
-// The token string must be 20 characters in length to be recognized as
-// a valid personal access token.
+// The token string must be 26 characters in length and have the 'glpat-'
+// prefix to be recognized as a valid personal access token.
 //
 // TODO: check if token has minimum scopes required by glab
 func IsValidToken(token string) bool {
-	return len(token) == 20
+    return len(token) == 26 && token[:6] == "glpat-"
```

Comment:
This would break older versions of self-hosted GitLab instances. 

We could still validate both newer and older versions:
```go
(len(token) == 26 && token[:6] == "glpat-") || len(token) == 20
```

Reply:
Also make sure to run gofmt on the change or you can run
`make fix`

Reply:
ok, i'll check it out, thx for the patience

I was looking at the tests right now and I was wondering if I should add one with the new token standard, guess I'll do that now